tests/tests.py

```diff
@@ -73,15 +78,18 @@ def test_login_returns_serialized_token(self):
         self.assertNotIn(username_field, response.data)
 
     def test_login_returns_serialized_token_and_username_field(self):
-        self.assertEqual(AuthToken.objects.count(), 0)
-        url = reverse('knox_login')
-        self.client.credentials(
-            HTTP_AUTHORIZATION=get_basic_auth_header(self.username, self.password)
-        )
-        knox_settings.USER_SERIALIZER = UserSerializer
-        response = self.client.post(url, {}, format='json')
+
+        with override_settings(REST_KNOX=user_serializer_knox):
```

Comment:
You could use the decorator as the override can be applied to the all test.

```
@override_settings(REST_KNOX=user_serializer_knox)
def test_login_returns_serialized_token_and_username_field(self):
...
```

This clarifies the fact that the settings can be overridden during the all lifetime of the test and also remove the indentation which makes the code slightly more readable.

This applies to the other usage below.
It's really just a cosmetic change, nothing blocking really.

Reply:
I disagree on that point, as we need to reload the module (here `views`) after the override as well. Having the global override would make this more difficult.